.github/dependabot.yml

```diff
@@ -3,8 +3,8 @@ updates:
   - package-ecosystem: 'npm'
     directory: '/'
     schedule:
-      interval: 'weekly'
+      interval: 'daily'
     allow:
       # Allow both direct and indirect updates for all packages
       - dependency-type: "all"
-    open-pull-requests-limit: 3
+    open-pull-requests-limit: 4
```

Comment:
The limit should probably be higher if there's a wait time before merge.

Reply:
Yes, but let's keep this in experimental mode for now. If we have to rebase all those all the time, it's better to have only a few :)

Reply:
I think dependabot might automatically rebase after some time. 🤔

Reply:
Only if there is a conflict AFAIK